logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/DefaultMinLogLevelIT.java

```diff
@@ -25,4 +44,37 @@ public void checkDefaultLogMinLevel() {
         // the value of minimum logging level overrides the logging level
         app.logs().assertDoesNotContain("Trace log example");
     }
+
+    // Verifies https://github.com/quarkusio/quarkus/issues/40016
+    @Test
+    public void checkLogRotationContent() throws IOException {
+        app.given().when().get("/log").then().statusCode(204);
+
+        Awaitility.await().atMost(10, TimeUnit.SECONDS).until(() -> {
+            File logDir = new File(LOG_DIR);
+            File[] rotatedFiles = logDir.listFiles((dir, name) -> name.startsWith(LOG_FILE_NAME));
+            return rotatedFiles != null && rotatedFiles.length > 1;
+        });
+
+        // Verify that the rotated file with the expected suffix exists
+        String expectedSuffix = "." + DATE_FORMATTER.format(LocalDate.now()) + "\\.\\d+$";
+        Path logDirPath = Path.of(LOG_DIR);
+        List<Path> rotatedFiles = Files.list(logDirPath)
+                .filter(path -> path.getFileName().toString().matches(LOG_FILE_NAME + expectedSuffix)).toList();
```

Comment:
I don't understand where is difference between your code and https://github.com/quarkus-qe/quarkus-test-framework/blob/main/quarkus-test-core/src/main/java/io/quarkus/test/services/quarkus/LocalhostQuarkusApplicationManagedResource.java#L47 that you can access with `new RestService().logs().assertContains()`. Can't you avoid all of this code or maybe leverage our framework somehow?

Reply:
What you address is our custom logging from FW. I test quarkus logging rotation to the file, that is by default named "quarkus.log"

Reply:
Isn't it in the same folder?

Reply:
I mean, if it is the same folder, same format, just differs in the file name, you can easily do what our FW do and you have log asserter. So `io.quarkus.test.utils.LogsVerifier` doesn't allow you to specify concrete filename, which makes it impossible. Ok.

Reply:
I made some simplification in code but the principle remains the same